website/content/docs/agent/telemetry.mdx

```diff
@@ -519,47 +519,94 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.grpc.server.streams`                        | Measures the number of active gRPC streams handled by the server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | streams                           | gauge   |
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | streams                           | gauge   |
 
+
+## Server Workload
+
+** Requirements: **
+* Consul 1.12.0+
+
+We added new RPC metrics as of 1.12 to better understand A) the workload on a Consul server and B) where that workload is coming from. These metrics give insight into that:
```

Comment:
How about:
"New RPC metrics were added as of Consul 1.12..."
and (2nd sentence)
"The following metric(s) provide that insight."

Reply:
Thanks for having a look matt!

This all looks good and I can follow up on it. About:

> "New RPC metrics were added as of Consul 1.12..."

I recall that we should use more active voice in our writing. But also curious to see what consul docs folks say

Reply:
You are 💯  correct @FFMMM. You're making me shed proud docs tears over here 🥲